frontend/app/routes/$lang+/_protected+/alerts+/manage+/edit.tsx

```diff
@@ -0,0 +1,186 @@
+import { useEffect, useMemo } from 'react';
+
+import type { ActionFunctionArgs, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
+import { json, redirect } from '@remix-run/node';
+import { useFetcher, useLoaderData } from '@remix-run/react';
+
+import { useTranslation } from 'react-i18next';
+import validator from 'validator';
+import { z } from 'zod';
+
+import pageIds from '../../../page-ids.json';
+import { Button } from '~/components/buttons';
+import { ErrorSummary, createErrorSummaryItems, hasErrors, scrollAndFocusToErrorSummary } from '~/components/error-summary';
+import { InputField } from '~/components/input-field';
+import { InputRadios } from '~/components/input-radios';
+import { getAuditService } from '~/services/audit-service.server';
+import { getInstrumentationService } from '~/services/instrumentation-service.server';
+import { getLookupService } from '~/services/lookup-service.server';
+import { getSubscriptionService } from '~/services/subscription-service.server';
+import { featureEnabled } from '~/utils/env.server';
+import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
+import { getFixedT } from '~/utils/locale-utils.server';
+import { getLogger } from '~/utils/logging.server';
+import { mergeMeta } from '~/utils/meta-utils';
+import { IdToken, UserinfoToken } from '~/utils/raoidc-utils.server';
+import { getPathById } from '~/utils/route-utils';
+import type { RouteHandleData } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
+
+export const handle = {
+  breadcrumbs: [{ labelI18nKey: 'alerts:manage.page-title' }],
+  i18nNamespaces: getTypedI18nNamespaces('alerts', 'gcweb'),
+  pageIdentifier: pageIds.protected.alerts.manage,
+  pageTitleI18nKey: 'alerts:manage.page-title',
+} as const satisfies RouteHandleData;
+
+export const meta: MetaFunction<typeof loader> = mergeMeta(({ data }) => {
+  return data ? getTitleMetaTags(data.meta.title) : [];
+});
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
```

Comment:
For all routes in the authenticated space, like this one, we need to add a `await raoidcService.handleSessionValidation(request, session);` in the `loader(..)`